### ISSUE TITLE
[Regression | 1.6 RC] multipass instances do not launch with cloud-init

### ISSUE THREAD

**Issue by joshti** [canonical/multipass#1919]
**Additional context**
We have thousands of users using Multipass through our program since December 2020. A user tried 1.6.0 for the storage relocation feature but then experienced no instances successfully launching using cloud-init. If you're not using cloud-init on 1.6.0, the instance launches successfully. We presently recommend users install 1.5.0 per our documentation, and this will become very problematic should 1.6.0 be released as-is.

**Describe the bug**
The 1.6 release candidate no longer results in a successful launch with cloud-init.

**To Reproduce**
1. `snap install multipass --beta`
1. `curl https://raw.githubusercontent.com/tigera/ccol1/main/control-init.yaml | multipass launch -n control -m 2048M 20.04 --cloud-init -`
1. `Invoke-WebRequest -UseBasicParsing 'https://raw.githubusercontent.com/tigera/ccol1/main/control-init.yaml' | Select-Object -Expand Content | multipass launch -n control -m 2048M 20.04 --cloud-init -`

**Expected behavior**
```
Name                    State             IPv4             Image
control                 Running           10.220.137.93    Ubuntu 20.04 LTS
```

**What happened instead**
```
Name                    State             IPv4             Image
control                 Unknown           --               Ubuntu 20.04 LTS
host1                   Unknown           --               Ubuntu 20.04 LTS
node1                   Restarting        --               Ubuntu 20.04 LTS
node2                   Unknown           --               Ubuntu 20.04 LTS
```

**Additional info**
- OS: Ubuntu 18.04 - 5.4.0
- `1.6.0-rc.262+g4c8f4ac3`
```
Name:           control
State:          Unknown
IPv4:           --
Release:        --
Image hash:     93cbee02d871 (Ubuntu 20.04 LTS)
Load:           --
Disk usage:     --
Memory usage:   --

Name:           host1
State:          Unknown
IPv4:           --
Release:        --
Image hash:     93cbee02d871 (Ubuntu 20.04 LTS)
Load:           --
Disk usage:     --
Memory usage:   --

Name:           node1
State:          Restarting
IPv4:           --
Release:        --
Image hash:     93cbee02d871 (Ubuntu 20.04 LTS)
Load:           --
Disk usage:     --
Memory usage:   --

Name:           node2
State:          Unknown
IPv4:           --
Release:        --
Image hash:     93cbee02d871 (Ubuntu 20.04 LTS)
Load:           --
Disk usage:     --
Memory usage:   --
```

cc @Saviq

**Comment by joshti** [comment]
The version that the user reported the issue on is `1.6.0-dev.259+g9c901b21`.

**Comment by Saviq** [comment]
Hi @joshti, thanks for letting us know, we'll look into it.

**Comment by Saviq** [comment]
Hi @joshti,

So the problem isn't that they don't launch "with cloud-init", but rather with your version of it. To enable bridging (#118) we had to modify how we set up networking in the instances, and that conflicted with how you're setting up the static IP.

#1913, however, makes it so that the change only applies _if_ you use the new feature, and we have it in queue to be added to 1.6.0, so your stuff should work fine again. I [shared the installers on that PR](https://github.com/canonical/multipass/pull/1913#issuecomment-760798221) for you to test.

All that said, your use case may be served even better by _using_ the new feature, with `--network id=<bridge>,mode=manual` (see [docs on the topic](https://multipass.run/docs/additional-networks)) you'll get a second interface that you can configure any way you want.

Let us know please how that works for you.

**Comment by Saviq** [comment]
I confirmed #1913 (now merged, coming to 1.6.0 RC soon) makes the instance launch with your `cloud-init.yaml`.

**Comment by joshti** [comment]
Thanks Saviq - I've confirmed the same on `1.7.0-dev.310.pr1913+g022dc80e`.
